packages/firestore/test/integration/api/database.test.ts

```diff
@@ -928,6 +928,58 @@ apiDescribe('Database', persistence => {
     });
   });
 
+  (persistence ? it : it.skip)(
+    'can clear persistence if the client has not been initialized',
+    async () => {
+      await withTestDoc(persistence, async docRef => {
+        await docRef.set({ foo: 'bar' });
+        const docSnap = await docRef.get({ source: 'cache' });
+        await expect(docSnap.data()).to.deep.equal({ foo: 'bar' });
+
+        const path = docRef.path;
+        const app = docRef.firestore.app;
+        const name = app.name;
+        const options = app.options;
+
+        // Verify data in cache persists after restart.
+        await app.delete();
+        const app2 = firebase.initializeApp(options, name);
+        const firestore2 = firebase.firestore!(app2);
+        await firestore2.enablePersistence();
+        const docRef2 = firestore2.doc(path);
+        const docSnap2 = await docRef2.get({ source: 'cache' });
+        expect(docSnap2.exists).to.be.true;
+        expect(docSnap2.data()).to.deep.equal({ foo: 'bar' });
```

Comment:
Consider removing this from the current test and creating a new test that verifies that persisted data is available after a restart.